apps/api/src/controllers/user.controller.ts

```diff
@@ -479,6 +479,97 @@ export function makeUserController(services: ServiceRegistry) {
         next(error);
       }
     },
+
+    /**
+     * Subscribe to Loops mailing list
+     * @param req Express request with userId from session
+     * @param res Express response
+     * @param next Express next function
+     */
+    async subscribe(req: Request, res: Response, next: NextFunction) {
```

Comment:
user auth gates the `/user/subscribe` and `/user/unsubscribe` endpoints. if the boolean `isSubscribed` conflicts with the intent, we throw an error; else, we update the user.